### PR TITLE
Backport: Update validations.js

### PR DESCRIPTION
**Pull request by Amir-61**
Add doc of allowNull options property per #372.

Related to https://github.com/strongloop/loopback-datasource-juggler/commit/9a1ef0849575a124f963ceb2043916b0b89fbf40#commitcomment-19250964

/to: @superkhau @crandmck

### CODE REVIEW

File: lib/validations.js [hunk 1]
```diff
@@ -127,9 +127,11 @@ Validatable.validatesNumericalityOf = getConfigurator('numericality');
  * ```
  *
  * @param {String} propertyName  Property name to validate.
- * @options {Object} Options
+ * @options {Object} Options See below
```
Comment (superkhau) [inline lib/validations.js]: Revert this line, see below is implied when blank.

**Comment by superkhau** [comment]
Can you also add a reference to the commit body? ie. Backport of https://github.com/strongloop/loopback-datasource-juggler/commit/9a1ef0849575a124f963ceb2043916b0b89fbf40 -- not sure how to deal with the line limit though. ;(

@bajtos Do you know of a way to shorten the URL so it fits within 72 chars?

**Comment by Amir-61** [comment]
@superkhau 

>Can you also add a reference to the commit body?

Didn't we decide to have the reference to original PR in commit description as optional?

Please see: https://github.com/strongloop-internal/loopback-knowledge-base/blob/master/backporting.md#how-to-backport

```
Optional: Referencing the original PR in the backport commit's description
```

AFAICT we decide to have the reference to the original PR in backporting PR as required not in backporting commit description.

**Comment by Amir-61** [comment]
@superkhau,

Please also see the conversation here: https://github.com/strongloop-internal/loopback-knowledge-base/pull/78

**Comment by superkhau** [comment]
@Amir-61 There was no PR that you're backporting for -- he commit directly to master remember?

**Comment by bajtos** [comment]
The merge commit landing this change will point to this pull request, which in turns point to the original (backported) commit. That's good enough as far as I am concerned.

> Can you also add a reference to the commit body? ie. Backport of https://github.com/strongloop/loopback-datasource-juggler/commit/9a1ef0849575a124f963ceb2043916b0b89fbf40 -- not sure how to deal with the line limit though. ;(

> @bajtos Do you know of a way to shorten the URL so it fits within 72 chars? Maybe "Backport of 9a1ef08"?

You can use the short SHA in the URL too, e.g.

```
Back-port 9a1ef08
https://github.com/strongloop/loopback-datasource-juggler/commit/9a1ef08
```

Using just the SHA without the URL is the second best option IMO.

```
Back-port 9a1ef08
```

**Comment by Amir-61** [comment]
@slnode test please

**Comment by superkhau** [comment]
@bajtos TY, gonna update our backporting doc with your suggestion.

**Comment by superkhau** [comment]
Updated note at https://github.com/strongloop-internal/loopback-knowledge-base/pull/110

**Pull request merged**